Clinical-Genomics/cgweb
# validate application for new samples

How to test:
0. install branch on clinical-db stage
1. login
2. new order
3. name="#123123" type=sequencing and scout, cust000
4. save
5. add family
6. add sample
7. type test as name, select first mip-test-sample
Expected outcome:
The sample should be selectable and savable without the OP complaining about invalid application tag
